Bottom: 5daf13607cae17ad811838c1d921252bedccae00
Top:    7d4e89e166d5add80cb36b620858bcf91ede81cb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 16:55:44 -0500

Refresh of add-buildbot-role-and-new

---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 78442f9f..e5fc5149 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -96,8 +96,8 @@ c['www'] = {
 # installation's html.WebStatus home page (linked to the
 # 'titleURL') and is embedded in the title of the waterfall HTML page.
 
-c['title'] = "workspace"
-c['titleURL'] = "http://wiki"
+c['title'] = "NovaTech CI"
+c['titleURL'] = "http://git.novatech-llc.com/"
 
 # the 'buildbotURL' string should point to the location where the buildbot's
 # internal web server (usually the html.WebStatus page) is visible. This
